go/libkb/rpc_exim.go

```diff
@@ -601,6 +601,17 @@ func ImportStatusAsError(g *GlobalContext, s *keybase1.Status) error {
 		}
 	case SCChatClientError:
 		return ChatClientError{Msg: s.Desc}
+	case SCChatUsersAlreadyInConversationError:
+		var cid chat1.ConversationID
+		var uids []keybase1.UID
+		for _, field := range s.Fields {
+			if field.Key == "conv_id" {
+				cid = chat1.ConversationID(field.Value)
```

Comment:
need to call `MakeConvID` to hex decode here, but I think you can just drop this field altogether